web/server/routers/post.ts

```diff
@@ -350,6 +356,12 @@ export const postRouter = router({
         houseTrained: { $in: houseTrained },
         spayNeuterStatus: { $in: spayNeuterStatus },
       };
+      let completeFilter;
+      if (input.covered != undefined) {
+        completeFilter = { ...noncoveredFilter, covered: input.covered };
+      } else {
+        completeFilter = noncoveredFilter;
+      }
```

Comment:
Sorry, I don't quite understand. Shouldn't the filter have the remaining values of the base filter? I did not directly add the covered to the base filter because that gives me a TypeError.